test/future_execution_test.rb

```diff
@@ -94,13 +105,13 @@ module FutureExecutionTest
       end
 
       describe 'serializers' do
-        let(:save_and_load) do
```

Comment:
Are we sure that the `serialized_args` won't be changed by saving/loading it to/from a database?